src/main/java/net/rubygrapefruit/platform/Native.java

```diff
@@ -1,99 +1,101 @@
-/*
- * Copyright 2012 Adam Murdoch
- *
- *    Licensed under the Apache License, Version 2.0 (the "License");
- *    you may not use this file except in compliance with the License.
- *    You may obtain a copy of the License at
- *
- *        http://www.apache.org/licenses/LICENSE-2.0
- *
- *    Unless required by applicable law or agreed to in writing, software
- *    distributed under the License is distributed on an "AS IS" BASIS,
- *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *    See the License for the specific language governing permissions and
- *    limitations under the License.
- */
-
-package net.rubygrapefruit.platform;
-
-import net.rubygrapefruit.platform.internal.NativeLibraryLoader;

```

Comment:
I suppose this file now also has different line endings. Shall we convert all files to LF on `master` in a separate PR?

Reply:
Damn. Yeah, let's convert. Would you mind taking that on?

Reply:
Done.